app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="statusBar">
+    <div>
+      <%= link_to image_tag(asset_path('back-arrow-icon.svg'), class: 'backIcon'), unauthenticated_root_path %>
+    </div>
+    <div>
+      <p class="statusBarTitle">
+    Sign Up
+    </p>
+    </div>
+    <div>
+    </div>
+    </div>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+<div class="formFieldsContainer">
+<div class="field">
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: "Full Name", class: 'formField' %>
+  </div>
+
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email", class: 'formField' %>
+  </div>
+
+  <div class="field">
+
+    <%= f.password_field :password, autocomplete: "new-password", placeholder: "Password", class: 'formField' %>
+  </div>
+
+  <div class="field">
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: "Confirm Password", class: 'formField' %>
+  </div>
+
+  </div>
+
+  <div class="newCategoryBtnContainer">
+    <%= f.submit "Sign up", class: "addCategoryBtnB" %>
+  </div>
+<% end %>
+
```

Comment:
- [ ] Good job with your app design it is good but I think you could make it better. I share with you some screenshots of the design that you can improve. Please make that all your app sticks to the original design as much as possible these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) and make sure that Colors, Typography: font face, size, weight, and Layout: composition and space between elements. :+1: 

#### Original design
![App1](https://user-images.githubusercontent.com/6615394/209413974-42e94dce-39ad-4d84-b5d4-a4faad243ea5.jpg)
![App2](https://user-images.githubusercontent.com/6615394/209413975-fc6dba85-7311-4d2e-a10c-44685d38322c.jpg)